src/main/java/com/stripe/net/StripeRequest.java

```diff
@@ -73,6 +78,7 @@ public StripeRequest(
       this.url = buildURL(method, url, params);
       this.content = buildContent(method, params);
       this.headers = buildHeaders(method, this.options);
+      this.usage = usage;
```

Comment:
How is this used?

Reply:
I see `request.usage()` in `sendWithTelemetry`

Reply:
Should we add a parameter to `requestWithTelemetry` instead? Feels dirty to leak usage into `StripeRequest` just to carry it around.

Reply:
Can you say more about the advantages of that? My current mental model is that the purpose of `StripeRequest` is to "carry around" values between `StripeResponseGetter.request` and `HttpClient.requestWithRetries` (and to provide a handle on this for tests) so adding another value doesn't feel dirty to me.

Reply:
FWIW I think maybe a problem here is that responsibility for setting telemetry would feel better living in `StripeResponseGetter` than in `HttpClient`. I'd prefer that not be in scope here, but I was able to restrict the scope of `StripeRequest.usage` to the `com.stripe.net` package in [Restrict StripeRequest.usage visibility](https://github.com/stripe/stripe-java/pull/1721/commits/27097f394c0a1102552d9b83bb13349682f3c651), to prevent a breaking change if we want to do that migration later.